### DISCUSSION TITLE
How to compile a 32-bit librdkafka on a 64-bit linux system ？

### DISCUSSION THREAD

**Discussion by VisonZh** [edenhill/librdkafka#3448] [Q&A]
My system version：
LSB Version:	:core-4.1-amd64:core-4.1-noarch:cxx-4.1-amd64:cxx-4.1-noarch:desktop-4.1-amd64:desktop-4.1-noarch:languages-4.1-amd64:languages-4.1-noarch:printing-4.1-amd64:printing-4.1-noarch
Distributor ID:	RedHatEnterpriseServer
Description:	Red Hat Enterprise Linux Server release 7.3 (Maipo)
Release:	7.3
Codename:	Maipo

**Comment by edenhill** [comment]
By passing your cross-toolchain CC, CPP, CXX, CPPFLAGS, CXXFLAGS, LDFLAGS, LIBS, LD, NM, etc, etc. See output from `./configure --help`

**Reply by VisonZh** [reply]
tks！
